lms/urls.py

```diff
@@ -609,9 +609,13 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-    urlpatterns += static(
-        settings.PROFILE_IMAGE_DOMAIN + settings.PROFILE_IMAGE_URL_PATH, document_root=settings.MEDIA_ROOT
-    )
+    # DO NOT MERGE this configuration should not be conditional, even in devstack/DEBUG,
+    # but it's here for the moment as it permits validating using the s3 backend in devstack.
+    if 'location' in settings.PROFILE_IMAGE_BACKEND_CONFIG['storage_kwargs']:
```

Comment:
yes, that was the idea

On Wed, Mar 25, 2015 at 10:11 AM, Daniel Friedman <notifications@github.com>
wrote:

> In lms/urls.py
> <https://github.com/edx/edx-platform/pull/7464#discussion_r27123219>:
>
> > @@ -609,9 +609,13 @@
> >
> >  if settings.DEBUG:
> >      urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
> > -    urlpatterns += static(
> > -        settings.PROFILE_IMAGE_DOMAIN + settings.PROFILE_IMAGE_URL_PATH, document_root=settings.MEDIA_ROOT
> > -    )
> > +    # DO NOT MERGE this configuration should not be conditional, even in devstack/DEBUG,
> > +    # but it's here for the moment as it permits validating using the s3 backend in devstack.
> > +    if 'location' in settings.PROFILE_IMAGE_BACKEND_CONFIG['storage_kwargs']:
>
> Do you mean that the PROFILE_IMAGE_BACKEND_CONFIG should always be
> configured for local FS storage in the case of DEBUG, making this check
> redundant?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7464/files#r27123219>.
>